media/server/gstplayer/include/GenericPlayerContext.h

```diff
@@ -46,6 +47,15 @@ struct Rectangle
     inline void clear() { x = y = width = height = 0; }
 };
 
+/**
+ * @brief Structure used for set source position
+ */
+struct SegmentData
+{
+    int64_t position;
```

Comment:
Should this be uint64_t ?

I might have spotted something :-)

Reply:
no, position has to be int64_t